src/api/forms/draft-form-definition-repository.js

```diff
@@ -38,6 +40,18 @@ export async function create(id, formDefinition) {
   await uploadToS3(formDefinitionFilename, formDefinitionString)
 }
 
+/**
+ * Copy the draft form to live in the Form Store
+ * @param {string} id - id
+ */
+export async function createLiveFromDraft(id) {
+  const draftDefinitionFilename = getFormDefinitionFilename(id)
```

Comment:
I know it's the default, but seeing it here makes it a lot more obvious

What do you think?

```suggestion
  const draftDefinitionFilename = getFormDefinitionFilename(id, 'draft')
```